README.rst

```diff
@@ -98,11 +98,10 @@ Examples
 -  `Extract and Visualize Individual Heartbeats <https://neurokit2.readthedocs.io/en/latest/examples/heartbeats.html>`_
 -  `Delineate QRS peaks, onsets and offsets <https://neurokit2.readthedocs.io/en/latest/examples/ecg_delineation.html>`_
 
-*You can try out these examples directly in your browser by* [***clicking here***](https://github.com/neuropsychology/NeuroKit/tree/master/docs/examples#cloud-based-interactive-examples).
+*You can try out these examples directly in your browser by clicking on the badge here*
```

Comment:
I think it's best to redirect them to the README here, reason is that if they know what's binder and all, they will know that they can click on the badge and will know what to expect. 

In the contrary case, they might benefit from first reading the documentation regarding the cloud-based interactive examples